### PR TITLE
[WIP]: Remove the bugzilla requirement for console feature work in 4.7

### PR DESCRIPTION
**Pull request by jcaianirh**
The console team is using the SLO pilot program where bugfix only sprints, and thus the bugzilla label, is not a requirement for the team to merge pr's.
The required merge label is blocking the repo merge process. For example: https://github.com/openshift/console/pull/7474

cc: @pweil-

### CODE REVIEW

**Comment by pweil-** [comment]
@bparees could you review this one?

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -825,8 +844,6 @@ tide:
     - openshift-priv/external-storage
     - openshift-priv/federation-v2-operator
     - openshift-priv/file-integrity-operator
-    - openshift-priv/gcp-pd-csi-driver
```
Comment (pweil-) [inline core-services/prow/02_config/_config.yaml:828]: these non-console related changes make me wonder if something isn't quite right yet.  :thinking:

**Comment by pweil-** [comment]
Actually, Ben is on PTO.

@wking think you can throw us an assist?  The goal here is that the Console team (among others) is piloting the "no feature freeze" effort and would like to remove the valid-bug requirement that is enforced post freeze so they don't have to make dummy bugs for features merges.  

I'm not confident enough to review this myself

File: core-services/prow/02_config/_config.yaml [hunk 2]
```diff
@@ -1785,6 +1784,7 @@ tide:
     - openshift/community
     - openshift/config
     - openshift/configure-alertmanager-operator
+    - openshift/console
```
Comment (petr-muller) [inline core-services/prow/02_config/_config.yaml:1787]: This change discards pretty much all OCP process-related (includinng the ones not related to BZs, like cherry-pick-approved ones) label requirements on *all* branches (including past release ones), which goes way beyong the intention stated in the PR description.

File: core-services/prow/02_config/_config.yaml (hunk 1 above)
Comment (petr-muller) [reply core-services/prow/02_config/_config.yaml:828]: The queries are just reordered (and they are similar so they switching places manifest as these small changes).

File: core-services/prow/02_config/_config.yaml (hunk 2 above)
Comment (petr-muller) [reply core-services/prow/02_config/_config.yaml:1787]: I thnk you need to do the following:

1. Remove openshift/console from the query that only covers `master` branch (you did this correclty in this PR)
2. Add an entirely new query that only covers `master` branch of `openshift/console` repo. In this PR you added the repo to an existing, relaxed-criteria query which is incorrect, I think you will need to add an entirely new query (basically make a copy of the one from where you are removing it, and 1) remove all other repos from it, 2) remove the valid-bz requirement from it).

Also, what are the expectattions when we do the cut? What criteria will hold for the `release-4.7` branch?

**Review by petr-muller — APPROVED** [review]

File: core-services/prow/02_config/_config.yaml (hunk 2 above)
Comment (pweil-) [reply core-services/prow/02_config/_config.yaml:1787]: We should retain all other process related controls for the branch and only remove the requirement for `valid-bug` labels post feature freeze on the master branch. Z-stream branches would still require bugs and the `valid-bug` label and we should follow the bug titling format for master even when not enforcing the label :+1:

**Comment by jcaianirh** [comment]
Closing and reopening a new pr with the above suggestions

**Comment by jcaianirh** [comment]
See https://github.com/openshift/release/pull/14476

**Pull request closed without merging**